torch/testing/_comparison.py

```diff
@@ -1007,28 +1021,10 @@ def _compare_regular_values_close(
             )
         else:
             msg = make_tensor_mismatch_msg(
-                actual, expected, ~matches, rtol=rtol, atol=atol, identifier=identifier
```

Comment:
In the error case, we created the `mismatches = ~matches` tensor here and turned it back into a `matches` inside `make_tensor_mismatch_msg`. With a minor refactoring, we no longer need to invert `matches` and can use it directly.